src/client/pages/Embark/ErrorModal.tsx

```diff
@@ -8,25 +8,43 @@ import { useCurrentLocale } from 'l10n/useCurrentLocale'
 import { Button } from 'components/buttons'
 import { useTextKeys } from 'utils/textKeys'
 import { ErrorModal, ErrorHeading, ErrorText } from 'components/ErrorModal'
+import { MEDIUM_SMALL_SCREEN_MEDIA_QUERY } from 'utils/mediaQueries'
 
 const { gray900, white } = colorsV3
 
 const ButtonContainer = styled.div`
   display: flex;
   justify-content: flex-start;
+  flex-wrap: wrap;
   margin-top: 1rem;
+  gap: 0.875rem;
 `
 
 const ButtonOutlined = styled(Button)`
   border: 1px solid ${gray900};
   background: ${white};
   color: ${gray900};
-  margin-left: 0.875rem;
+  padding: 0.375rem 0.75rem;
+  font-size: 0.875rem;
```

Comment:
nit: maybe we could add this as a `sm` prop to button later on? :)

Reply:
Yes, we have some completely new button components coming along, but until then I decided to just do this tweak :)